shark/shark_runner.py

```diff
@@ -49,18 +50,34 @@ def __init__(
         func_name = "forward"
         device = device if device is not None else shark_args.device
         if self.frontend in ["pytorch", "torch"]:
+            #get torch-mlir dialet
```

Comment:
NIT: typo + can we have a space, as in `# get torch-mlir dialect`

Reply:
Done